gusto/physics/held_suarez_forcing.py

```diff
@@ -0,0 +1,173 @@
+        self.exner_interpolator = Interpolator(
+            thermodynamics.exner_pressure(equation.parameters,
+                                          self.rho_averaged, self.theta), self.exner)
+        self.sigma = Function(Vt)
+        self.kappa = equation.parameters.kappa
+        HS_params = Held_Suarez_Parameters
+
+        theta_condition = (HS_params.T0surf - HS_params.T0horiz * sin(lat)**2 - (HS_params.T0vert * ln(self.exner) * cos(lat)**2)/self.kappa)
+        Theta_eq = conditional(HS_params.T0stra/self.exner >= theta_condition, HS_params.T0stra/self.exner, theta_condition)
+
+        # timescale of temperature forcing
+        tau_cond = (self.sigma**(1/self.kappa) - HS_params.sigmab) / (1 - HS_params.sigmab)
+        newton_freq = 1 / HS_params.tau_d + (1/HS_params.tau_u - 1/HS_params.tau_d) * conditional(0 >= tau_cond, 0, tau_cond) * cos(lat)**4
+        forcing_expr = newton_freq * (self.theta - Theta_eq)
+
+        # Create source for forcing
+        self.source_relaxation = Function(Vt)
+        self.source_interpolator = Interpolator(forcing_expr, self.source_relaxation)
+
+        # Add relaxation term to residual
+        test = equation.tests[theta_idx]
+        dx_reduced = dx(degree=4)
+        forcing_form = test * self.source_relaxation * dx_reduced
+        equation.residual += self.label(subject(prognostic(forcing_form, 'theta'), X), self.evaluate)
+
+    def evaluate(self, x_in, dt):
+        """
+        Evalutes the source term generated by the physics.
+
+        Args:
+            x_in: (:class:`Function`): the (mixed) field to be evolved.
+            dt: (:class:`Constant`): the timestep, which can be the time
+                interval for the scheme.
+        """
+        self.X.assign(x_in)
+        self.rho_recoverer.project()
+        self.exner_interpolator.interpolate()
+
+        # Determine sigma:= exner / exner_surf
+        exner_columnwise, index_data = self.domain.coords.get_column_data(self.exner, self.domain)
+        sigma_columnwise = np.zeros_like(exner_columnwise)
+        for col in range(len(exner_columnwise[:, 0])):
+            sigma_columnwise[col, :] = exner_columnwise[col, :] / exner_columnwise[col, 0]
+        self.domain.coords.set_field_from_column_data(self.sigma, sigma_columnwise, index_data)
+
+        self.source_interpolator.interpolate()
+
+
+class RayleighFriction(PhysicsParametrisation):
+    """
+    Forcing term on the velocity of the form
+    F_u = -u / a,
+    where a is some friction factor
+    """
+    def __init__(self, equation, parameters=None):
+        """
+         Args:
+            equation (:class:`PrognosticEquationSet`): the model's equation.
+            forcing_coeff (:class:`unsure what to put here`): the coefficient
+            determining rate of friction
+        """
+        label_name = 'rayleigh_friction'
+        super().__init__(equation, label_name, parameters=parameters)
+        self.parameters = equation.parameters
```

Comment:
Can you add some new lines at appropriate places in this block of code to make it more readable?